wingtips-core/src/test/java/com/nike/wingtips/http/HttpRequestTracingUtilsTest.java

```diff
@@ -260,6 +260,51 @@ public void fromRequestWithHeaders_sets_sampleable_to_attribute_value_if_sampled
         assertThat(newSpan.isSampleable()).isFalse();
     }
 
+    @DataProvider(value = {
```

Comment:
You didn't read the umpteen bazillion tests with a fine toothed comb?? For shame! ;)